qutebrowser/mainwindow/prompt.py

```diff
@@ -630,6 +630,18 @@ def __init__(self, question, parent=None):
 
         self._to_complete = ''
 
+    def _directories_hide_show_model(self, path):
+        """Get rid of non-matching directories."""
+        try:
+            num_rows = self._file_model.rowCount(self._root_index)
+            for row in range(num_rows):
+                index = self._file_model.index(row, 0, self._file_model.index(path))
+                hidden = self._to_complete not in index.data()
+                self._file_view.setRowHidden(index.row(), index.parent(), hidden)
+        except FileNotFoundError:
+            log.prompt.debug("Directory doesn't exist, can't \
+                                hide and unhide file prompt folders")
```

Comment:
nitpick: Please use Python's implicit line joining (inside braces) instead, and make sure the second line is aligned with the first one:

```suggestion
            log.prompt.debug("Directory doesn't exist, can't "
                             "hide and unhide file prompt folders")
```